build/primary/main/locals.tf

```
@@ -14,4 +14,5 @@
 locals {
   DATA_LAKE_TIMESERIES_UNPROCESSED_BLOB_NAME    = "timeseries-unprocessed"
   DATA_LAKE_DATA_CONTAINER_NAME                 = "timeseries-data"
+  TIME_SERIES_RAW                               = "timeseries-raw"
```

How about? (And if yes, then DATA_LAKE_TIMESERIES_UNPROCESSED_BLOB_NAME should probably be updated similarly)

```suggestion
  DATA_LAKE_TIME_SERIES_RAW_BLOB_FOLDER                               = "timeseries-raw"
```

---

renamed